### DISCUSSION TITLE
Fix different icon variant sizing on buttons

### DISCUSSION THREAD

**Discussion by eckelarsson** [livewire/flux#548] [Feature Requests] [answered]
Hi,

**Flux button with icon variant**
I notice when I source dived that thier is an icon variant on the flux button, I tried to change it to outline but then the icon sizes and placement just went a little bonkers.

![Image](https://github.com/user-attachments/assets/f800e6d6-9ba0-4fa1-8330-ede7e99cbdaa)

**Comment by jeffchown** [comment]
@eckelarsson I tried buttons with each `icon-variant` and it looks like any `icon-variant` **except** micro throws off the button's styling:

```
<div class="space-x-2">
    <flux:button icon="arrow-down-circle">No icon-variant</flux:button>
    <flux:button icon="arrow-down-circle" icon-variant="outline">Outline</flux:button>
    <flux:button icon="arrow-down-circle" icon-variant="solid">Solid</flux:button>
    <flux:button icon="arrow-down-circle" icon-variant="mini">Mini</flux:button>
    <flux:button icon="arrow-down-circle" icon-variant="micro">Micro</flux:button>
</div>
```

![Image](https://github.com/user-attachments/assets/3f630257-70c8-435c-88bb-09357ce9f372)

**Comment by calebporzio** [comment]
In general, the reason is, is that each heroicon variant was designed to be used at it's designed size. Otherwise, the strokes in the outline variant get too thin. Also you can into sub-pixel aliasing issues as well, making them not look as crisp:

![Image](https://github.com/user-attachments/assets/60a5037e-acc8-4b5f-8daa-9690eadd5eee)

You can always include the icon in the slot by hand if you need downsized outline icons.

I'm going to keep this in mind as a feature request instead of an issue for now. thanks

**Comment by joshhanley** [comment, marked as answer]
This has been fixed in PR #987 and released in v1.1.3

**Reply by eckelarsson** [reply]
🙌🏼🙌🏼🙌🏼
